monitor/xfeemngr/contract/bound.go

```diff
@@ -87,6 +88,7 @@ func (c BoundFeeOracleV1) SetGasPriceOn(ctx context.Context, destChainID uint64,
 		return errors.Wrap(err, "tx opts")
 	}
 
+	log.Info(ctx, "Setting gas price on chain", "destination_chain_id", destChainID, "rate", gasPrice.Int64())
```

Comment:
Make it easy for humans to read logs: best use proper names instead of chainIDs (see `netconf.Network.ChainName`). Also just use `dest_chain` for consistency with rest of code base. Also maybe convert wei into gwei.